### PR TITLE
Updating Aditya's personal page

### PR DESCRIPTION
**Pull request by aditya-uw**
Was not sure what news items to add during our group meeting on Oct 17th so I updated some words here and there in my bio.

Anyone can look and let me know how they feel! Or let me know if I should merge :)

### CODE REVIEW

**Comment by aditya-uw** [comment]
On second thought, let me see if I have a CV somewhere...

**Comment by aditya-uw** [comment]
Okay I added my CV so now I'm good on my end!

File: content/authors/adkris/_index.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ organizations:
 - name: Applied Physics Lab
 
 # Short bio (displayed in user profile at end of posts)
-bio: I am interested in applying machine learning algorithms for understanding echolocating animals
+bio: I am interested in studying echolocation as a sensory tool.
```
Comment (leewujung) [inline content/authors/adkris/_index.md:23]: "Sensor tool" is a little odd. What do you mean to say?

File: content/authors/adkris/_index.md [hunk 2]
```diff
@@ -67,9 +67,9 @@ user_groups:
 
 I am a student at the University of Washington majoring in Electrical Engineering and pursuing Digital Signal Processing and Neural Engineering concentrations. I am expected to graduate the Spring of 2024 and am interested in pursuing graduate research after my bachelor's degree.
 
-My core competency is in programming, with an expertise in Python and Java. I am currently applying signal processing techniques in passive acoustic monitoring of bats with a goal toward understanding echolocation as a sensory tool.
+My core competency is in signal processing, with extensive background using Python. I enjoy interdisciplinary research and find that bats and whales are fascinating model systems to study due to their ability to echolocate. I am currently applying signal processing techniques in passive acoustic monitoring of bats with a goal toward understanding echolocation as a sensory tool.
```
Comment (leewujung) [inline content/authors/adkris/_index.md:70]: small wording suggestion:
```suggestion
My core competency is in signal processing, with extensive experience programming in Python. I enjoy interdisciplinary research and find that bats and whales are fascinating model systems to study due to their ability to echolocate. I am currently applying signal processing techniques in passive acoustic monitoring of bats with a goal toward understanding echolocation as a sensory tool.
```

File: content/authors/adkris/_index.md (hunk 1 above)
Comment (aditya-uw) [reply content/authors/adkris/_index.md:23]: I guess that part is also redundant but I wanted to explain that I am interested in studying how animals echolocate. How is that "tool" of echolocation being used by bats, for example, for their tasks

Comment (leewujung) [reply content/authors/adkris/_index.md:23]: I think "sensing tool" is more broadly used to describe this.

Comment (aditya-uw) [reply content/authors/adkris/_index.md]: I like "sensing tool" thanks!

**Comment by leewujung** [comment]
LGTM -- Feel free to merge!

**Comment by aditya-uw** [comment]
Oh haha I don't have write access!

**Pull request merged**